src/test/e2e/39_crane_to_oras_test.go

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package test provides e2e tests for Zarf.
+package test
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/zarf-dev/zarf/src/pkg/utils/exec"
+)
+
+func TestCraneToORAS(t *testing.T) {
```

Comment:
A relatively novel test, this pulls down the latest Zarf version using Crane and makes sure that packages built with Crane still work with ORAS and vice versa.